iOSDFULibrary/Classes/Implementation/Firmware/DFUFirmware.swift

```diff
@@ -197,23 +205,25 @@ import Foundation
         let hex = ext.caseInsensitiveCompare("hex") == .orderedSame
         guard bin || hex else {
             NSLog("\(fileName!) is not a BIN or HEX file")
```

Comment:
Actually, with those new error handlers we may no longer need logging to `NSLog` anywhere in the file. This may be done by the app.